packages/loader/src/gltf/extensions/MeshoptDecoder.ts

```diff
@@ -27,10 +35,34 @@ const MeshoptDecoder = (function () {
 
   let instance;
 
-  const ready = WebAssembly.instantiate(unpack(wasm), {}).then(function (result) {
-    instance = result.instance;
-    instance.exports.__wasm_call_ctors();
-  });
+  ready = WebAssembly.instantiate(unpack(wasm), {})
+    .then(function (result) {
+      instance = result.instance;
+      instance.exports.__wasm_call_ctors();
+    })
+    .then(() => ({
+      workerCount: 4,
+      ready: ready,
+      useWorkers: function () {
+        initWorkers(this.workerCount);
+      },
+      decodeGltfBuffer: function (count, stride, source, mode, filter): Promise<Uint8Array> {
+        if (this.workerCount > 0 && workers.length === 0) this.useWorkers();
```

Comment:
Refer to KTX2 Loader default value and add a default value?

Reply:
The meshopt decoder is nested in the gltf loader. You mean that we should pass the exact parameter from gltf loader?